### ISSUE TITLE
When executed under PowerShell nuget.exe cannot find MSBuild

### ISSUE THREAD

**Issue by sandersaares** [NuGet/Home#1400]
Using nuget.exe version 3.2.0.10488 on Windows 10, I get the following error when I execute a "nuget pack" command from PowerShell:

```
Cannot find the specified version of msbuild: '4.6.79.0'
```

The operation works fine if I execute the same command from a Command Prompt window. To launch PowerShell, I simply run it from the start menu.

Typing "msbuild" into PowerShell results in MSBuild 4.6.79.0 successfully starting.

**Comment by yishaigalatzer** [comment]
The detection logic is based on installed msbuild version and what the toolset provides.

Can you tell us what msbuild returns when you run it from command line?

You can use an msbuild version picker -MSBuildVersion 4/12/14 depending on what project you are building. See http://docs.nuget.org/consume/Command-Line-Reference.

We can take a look at improving the auto detection in your scenario going forward, but note that now with multiple versions of msbuild available, you might want to use the flag when you script to allow for various things running on the path.

**Comment by sandersaares** [comment]
This is what I get when I run MSBuild from PowerShell:

```
C:\Users\saares> msbuild
Microsoft (R) Build Engine version 4.6.79.0
[Microsoft .NET Framework, version 4.0.30319.42000]
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.
```

It is likely loaded from C:\Windows\Microsoft.NET\Framework\v4.0.30319 as that is in my path variable under PowerShell.

From the command prompt, I get nothing because MSBuild is not in my path variable outside PowerShell.

Manually specifying MSBuildVersion works fine as a workaround, although it is an annoyance since I do not use any MSBuild related features in NuGet - I don't really care what version it detects or not. I am just using it as a glorified zip file maker, so rejecting the operation by not being able to find an unneeded app is just a kick in the face.

**Comment by yishaigalatzer** [comment]
Can you please update to 3.2.0.10516 from http://dist.nuget.org/index.html ? I believe we fixed the issue, but I'd like to verify it works in your scenario

**Comment by sandersaares** [comment]
I can confirm the issue is fixed in 3.2.0.10516. Thanks!
